com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ResourceEncryption.java

```diff
@@ -35,25 +35,19 @@
 
 	private static ResourceEncryptionPlugin encryptionPlugin;
 
+	private static DefaultResourceEncryption defaultEncryption = new DefaultResourceEncryption();
+
 	/**
 	 * Encrypt a resource
 	 * @param resource Bytes of resource data to encrypt
 	 * @return Bytes of encrypted resource data
 	 */
 	public static byte[] encrypt(byte[] resource) throws CompileExceptionError {
 		try {
-			if (encryptionPlugin == null) {
-				// find and create plugin instances
-				List<ResourceEncryptionPlugin> encryptionPlugins = PluginScanner.createPlugins("com.dynamo.bob.archive", ResourceEncryptionPlugin.class);
-				
-				// only accept a single encryption instance
-				if (encryptionPlugins.size() > 1) {
-					throw new CompileExceptionError("Found more than one ResourceEncryptionPlugin");
-				}
+			ResourceEncryptionPlugin encryptionPlugin = PluginScanner.createPlugin("com.dynamo.bob.archive", ResourceEncryptionPlugin.class);
```

Comment:
Same comment as previous PR, I don't like the code flow of a "create" for each resource.
It's not obvious that it's a get-or-create type of function.
We don't want to encourage people to sprinkle resource creation in the middle of the code.